docs/offset_function.rst

```diff
@@ -21,28 +33,30 @@ The code base uses a *dict* named *theta_prior* to handle the
 offset function.  It currently holds three items:
 
 * method -- A string defining the method used.  core, uniform, exp
-* max -- The maximum separation :math:`{\theta_{max}}`
-  in units of angular size.  :math:`p(w|O) = 0` for
+* max -- :math:`{\theta_{max}}`
+  in units of :math:`\phi`.  :math:`p(\omega|O) = 0` for
   :math:`\theta > \theta_{max}`
 * ang_size -- An array of angular sizes :math:`\phi`
   for all of the galaxies.
```

Comment:
minor detail: "for all of the galaxies" -> "for all the candidate galaxies" or "for all the potential host galaxies"?

Reply:
edited